docs/scos/user/features/202108.0/promotions-discounts-feature-overview.md

```diff
@@ -125,93 +125,258 @@ There are two types of discount application:
+* Voucher code discount with priority 3
+* Cart rule discount with priority 4
+
+In this case, the discounts are applied in the order of priority, irrespective of the type.
+
+{% info_block infoBox "Info" %}
+
+If you have a voucher code and a cart rule with the same priority, then both discounts should apply and use the same cart subtotal. See [Scenario 5: Non-exclusive, fixed amount and percentage discounts with the same priority values](#scenario-5-non-exclusive-fixed-amount-and-percentage-discounts-with-the-same-priority-values) for an example.
+
+{% endinfo_block %} 
+
+## Discount calculation logic
+
+The discount calculation logic follows these rules:
+
+* All discounts are calculated sequentially based on their [priority value](#discount-priority). Thus, discounts with the highest priority value are considered first.
+* Discounts without priority values are treated with the last possible priority.
+* Discounts with the same priority value are calculated independent of each other.
+* If there is a set of discounts with [exclusive](#exclusive-discount) and [non-exclusive](#non-exclusive-discount) discounts, the non-exclusive discounts are disregarded. The remaining exclusive discounts are handled according to their priority values as shown in the schema:
+
+![discounts-calculation-logic](https://confluence-connect.gliffy.net/embed/image/ad6c6c4c-9ccb-42ae-a9dc-5944300bdf91.png?utm_medium=live&utm_source=custom)
+
+<!---source: https://spryker.atlassian.net/wiki/spaces/DOCS/pages/3019079947/CC-14560+-+Enable+prioritization+for+discounts -->
+
+### Example discount calculation scenarios
+
+The following scenarios illustrate how the discount calculation logic works.
+
+#### Scenario 1: Multiple non-exclusive discounts, mix of fixed amount and percentage discount calculation types.
+
+Cart subtotal: €500
+
+| DISCOUNT NAME|  DESCRIPTION|  DISCOUNT TYPE| DISCOUNT AMOUNT| DISCOUNT PRIORITY| EXCLUSIVENESS|
+| --- | --- | --- | --- | --- |--- |
+| HOCKEY10| 10% off your order| Percentage| 10%|  300|  No|
+| HELMET20| Save €20 on helmets| Fixed amount| €20|  200|  No|
+| STICK50| €50 off all carbon sticks| Fixed amount| €50|  500|  No|
+
+**Discounts applied in priority order:**
+
+1. HELMET20 (200 priority): €500 - €20 = €480
+2. HOCKEY10 (300 priority): €480 - €480*0.10 = €432
+3. STICK50 (500 priority): €432 - €50 = €382
+
+**Calculation as displayed in cart:**
+
+Subtotal: €500.00
+HELMET20: -€20.00
+HOCKEY10: -€48.00
+STICK50: -€50.00
+
+Grand total: €382.00
+
+{% info_block infoBox "Info" %}
+
+The assigned priorities can be confusing for customers, as they are not aware of the prioritization. For the scenario above, they might wonder why they get 10% off the pre-discount price of the hockey stick and not, for example, the helmet. Therefore, in the cases like this one, rather than creating prioritization logic for separate discount types, consider adjusting the discount prioritization accordingly to avoid customer confusion.
```

Comment:
```suggestion
As customers are not aware of the prioritization, they may be confused about the assigned priorities. For the scenario above, they might wonder why they get 10% off the pre-discount price of the hockey stick and not, for example, the helmet. Thus, instead of creating separate prioritization logic for discount types, consider adjusting the discount priority accordingly in cases such as this one to avoid customer confusion.
```